_docs/technical/2017-02-16-delegation.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: Stake Delegation in Cardano SL
+permalink: /technical/delegation/
+group: technical
+visible: true
+---
+[//]: # (Reviewed at e1d0f9fb37a3f1378341716916f0321fb55698df)
+
+# Stake Delegation in Cardano SL
+
+This guide describes implementation details of the stake delegation process.
+
+As described earlier, stakeholders must be online in order to generate new blocks when they are selected as slot-leaders.
+However, such a situation can be unattractive, because of requiring that a majority of elected stakeholders participate in the
+Coin Tossing protocol for refreshing randomness (crucial attribute of the slot-leader election process). But if we have a
+lot of elected stakeholders, such a requiring can introduce a strain on the stakeholders and the network, since it might
+require broadcasting and storing a large number of commitments and shares.
+
+Delegation feature allows stakeholders `S1...Sn` to transfer their "committee participation" to some delegates `D1...Dm`.
+These delegates will represent stakeholders `S1...Sn` in the Coin Tossing protocol. In this case the actual number of 
+nodes participating in the Coin Tossing protocol can be much lower, see [paper](/glossary/#paper), page 38.
+
+## Schema
+
+Slot-leader can transfer its right to generate new block to the delegate. To do it, slot-leader use _delegation-by-proxy_
+scheme: slot-leader generates [_proxy signing key_](https://github.com/input-output-hk/cardano-sl/blob/66efe143138e3b7cfb11373c05022e3b7da67d87/core/Pos/Crypto/SignTag.hs#L34), or PSK, and delegate will use it [to sign](https://github.com/input-output-hk/cardano-sl/blob/66efe143138e3b7cfb11373c05022e3b7da67d87/src/Pos/Delegation/Methods.hs#L54) messages to authenticate a block. There're two kinds of PSKs, heavyweight and lightweight (see below).
+
+Specifically, stakeholder forms a special certificate specifying the delegates identity via its public key.
+So later the delegate can sign messages within the valid message space by providing signatures for these messages
+under its own public key along with the signed certificate.
+
+This is format of a [proxy signature](https://github.com/input-output-hk/cardano-sl/blob/66efe143138e3b7cfb11373c05022e3b7da67d87/core/Pos/Crypto/Signing.hs#L309). It includes:
+
+1. Omega value,
+2. Delegate's public key,
+3. Proxy certificate (mentioned above),
+4. Signature.
+
+Omega (or ω) is a special value from the [paper](/glossary/#paper).
```

Comment:
Add a reference to where `omega` is first defined, forcing the reader to skim the paper to search for it is kind of mean, isn't it? 😂